src/ui/zcl_abapgit_gui_page_sett_locl.clas.abap

```diff
@@ -131,6 +136,11 @@ CLASS zcl_abapgit_gui_page_sett_locl IMPLEMENTATION.
       iv_name        = c_id-display_name
       iv_label       = 'Display Name'
       iv_hint        = 'Name to show instead of original repo name (optional)'
+    )->text(
+      iv_name        = c_id-transport_request
+      iv_side_action = c_event-choose_transport_request
+      iv_label       = |Transport request|
+      iv_hint        = 'Transport request; All changes are recorded therein and no transport popup appears|'
```

Comment:
The field should be hidden for local packages 
`zcl_abapgit_factory=>get_sap_package( iv_package )->are_changes_recorded_in_tr_req( ).`

Reply:
Thx. fixed.